src/index.js

```diff
@@ -0,0 +1,132 @@
+/* eslint-disable no-unused-vars */
+
+/* Import section */
+import _, { functionsIn, iteratee, update } from 'lodash';
+import './style.css';
+import taskcompleted from './status';
+
+/* Variable section */
+const ul = document.getElementById('list');
+let allTasks = [];
```

Comment:
At the very start of your code you could initialize `let allTasks = JSON.parse(localStorage.getItem('data')) || []`. What it does is the `JSON.parse(localStorage.getItem('data'))` see's if you had anything stored in your local storage and if there was something stored it will assign it to `let allTasks` else the `[]` will intialize your `let allTasks` with an empty array. 
If you want to learn more about it click [here](https://medium.com/infancyit/kiss-keep-it-simple-smart-javascript-habits-b019ce14b5ec)